test/framework/src/main/java/org/elasticsearch/repositories/ESBlobStoreContainerTestCase.java

```diff
@@ -112,6 +112,22 @@ public void testMoveAndList() throws IOException {
         }
     }
 
+    public void testDeleteBlob() throws IOException {
```

Comment:
Thanks! :smile: I should note that this test passes for all of the `BlobContainer` classes that were already being tested as well as Google.